### ISSUE TITLE
Diff Header color in master

### ISSUE THREAD

**Issue by vbjay** [gitextensions/gitextensions#5435]
**Do you want to request a *feature* or report a *bug*?**
feature
**What is the current behavior?**
![image](https://user-images.githubusercontent.com/3629489/45588174-408c7f00-b8de-11e8-806c-79d68acc7a74.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
1 of 2 options:

1. Use a formula to calculate the right font color based on the current background color.  https://stackoverflow.com/questions/1855884/determine-font-color-based-on-background-color
2. Allow user to pick font colors too.  

**Environment you encounter the issue:**

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 

**Did this work in previous version of GitExtensions (which)?**

**Comment by vbjay** [comment]
Are you picking aqua?  
![image](https://user-images.githubusercontent.com/3629489/45590833-60d53180-b910-11e8-854d-22b2f53753c4.png)

**Comment by mstv** [comment]
You're right, I chose the slightly brighter cyan from the first row which is ff80ffff instead of aqua with ff00ffff.

**Comment by RussKie** [comment]
How did you set this color?
I have deleted `C:\Users\<user>\AppData\Roaming\GitExtensions\GitExtensions\GitExtensions.settings` file, started GE and the default color is `ffe6e6e6`

**Comment by vbjay** [comment]
Go to colors page in settings. Last color option.

**Comment by RussKie** [comment]
I know where it is, thank you :)
My question was mainly about whether you have set that color deliberately, or it was set somehow else.
If it is something users sets expressly than that's their own choosing/fault.

**Comment by vbjay** [comment]
Yes. I set it and noticed the issue. Cyan works for now. I noticed the
difference when I ran both versions.

**Comment by vbjay** [comment]
Look at the screenshot at the bottom of the issue. It worked in 2.51.5.
It's a bug.

**Comment by RussKie** [comment]
It didn't register with me first time I looked at the issue.

The font color is white, and it is hard to read when the background color is set to a light color. Looks like the color diff algorithm isn't working correctly or isn't used at all.

**Comment by vbjay** [comment]
I posted a link to a stack overflow that shows a good algorithm.
